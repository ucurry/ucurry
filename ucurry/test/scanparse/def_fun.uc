-- def -> fun
fun: int -> int:
f x = x + 1;