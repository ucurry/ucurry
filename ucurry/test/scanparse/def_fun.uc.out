Printing Roundtrip
We only pretty print expressions right now

